pack/Microsoft.Azure.Functions.V2.PythonWorker.nuspec

```diff
@@ -20,7 +20,7 @@
     <file src="..\3.7_OSX_X64\**" target="tools\3.7\OSX\X64" />
     <file src="..\python\prodV2\worker.config.json" target="tools" />
     <file src="Microsoft.Azure.Functions.PythonWorker.targets" target="build" />
-    <file src="..\PythonWorker\_manifest\manifest.json" target="SBOM\manifest.json" />
-    <file src="..\PythonWorker\_manifest\spdx_2.2\manifest.spdx.json" target="SBOM\manifest.spdx.json" />
+    <file src="..\_manifest\manifest.json" target="SBOM\manifest.json" />
```

Comment:
For v4
https://github.com/Azure/azure-functions-python-worker/blob/117b8650a5a639e7e374d06352a9679ec7052815/pack/Microsoft.Azure.Functions.V4.PythonWorker.nuspec#L31

Reply:
for V3
https://github.com/Azure/azure-functions-python-worker/blob/117b8650a5a639e7e374d06352a9679ec7052815/pack/Microsoft.Azure.Functions.V3.PythonWorker.nuspec#L32